### DISCUSSION TITLE
upgrading

### DISCUSSION THREAD

**Discussion by laggysec** [projectdiscovery/nuclei-templates#694] [Q&A] [answered]
i compile new version and i moved to /usr/bin when i run nuclei its still showing 2.1.1 not 2.2.1-dev

**Comment by geeknik** [comment, marked as answer]
I believe this is because the old binary in your GOBIN directory still exists and takes precedence in your path, you will have to remove the old binary

**Reply by laggysec** [reply]
Thank you 
My bad i didnt change permission as well delete old one fro gobin

**Reply by geeknik** [reply]
You’re welcome, I ran into this very issue on Friday
